4.analyze_data/README.md

```diff
@@ -2,10 +2,10 @@
 
 In this module, we perform analysis on the data to uncover answers to our goals as specified in [the main README](../README.md).
 
-
 The analyses we perform are:
 
 1. Linear modeling per feature in each plate. The beta coefficients for WT treatment contribution and cell count contribution are plotted as a scatter plot.
+2. UMAP generation to assess if there are similar morphologies based on various metadata.
```

Comment:
"...based on various metadata." Could expand on this?
Metadata being the image or cell metadata? or the UMAP space of the morphology features? Or is the UMAP space on a coded metadata?

Reply:
I mean that we use different metadata per single cell to label the UMAPs, including genotype, cell count, etc. 

I can try to make this more explicit so thanks for bringing it up!